project/MimaBuild.scala

```diff
@@ -91,8 +91,8 @@ object MimaBuild {
 
   def mimaSettings(sparkHome: File, projectRef: ProjectRef) = {
     val organization = "org.apache.spark"
-    val previousSparkVersion = "1.2.0"
-    val fullId = "spark-" + projectRef.project + "_2.10"
+    val previousSparkVersion = "1.3.0"
+    val fullId = "spark-" + projectRef.project + "_2.11"
```

Comment:
Shouldn't this be controlled by the execution flags (or `change-version-to-2.11.sh`)? Note that the validator runs the `2.10` build but mima seems to look at 2.11.